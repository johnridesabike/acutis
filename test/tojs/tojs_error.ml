open Acutis

let () =
  let src =
    {|
  {% match bool with false %} {% /match %}
  {% match int_enum with @1 %} {% with @2 %} {% /match %}
  {% match string_enum with @"a" %} {% with @"b" %} {% /match %}
  {% match tuple with (a, b) %} {{ a }} {{ b }} {% /match %}
|}
  in
  ToJs.pp Format.std_formatter
    Compile.(from_string_nolink ~fname:"<test>" Components.empty src)
