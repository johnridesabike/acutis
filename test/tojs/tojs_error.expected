This field must be a boolean.
This field must be an int enum.
This field must be a string enum.
This field must be an array.
